src/interface/mesh_data.hpp

```diff
@@ -76,12 +77,23 @@ class MeshData {
   MeshData() = default;
 
   Mesh *GetMeshPointer() const { return pmy_mesh_; }
+  auto GetGridPointer() { return GetMeshPointer(); }
```

Comment:
it _does not_ point to the data.  it points to either `Mesh` or `MeshBlock`.  That's why I went with `Grid` because we hadn't gone there yet but has the closest meaning intuitively I think.

Reply:
Hmm.  OK.  I'll go with `GetParentPointer`, but I wish the name gave some sense of what sort of "thing" the parent is.  Can't think of anything though.